### ISSUE TITLE
Link to other ticket

### ISSUE THREAD

**Issue by KorvinEC** [pluginsGLPI/formcreator#2354]
Hello! I have a problem with target Ticket. My task is to be able to make form ticket a son (or be linked to) other, already created ticket. Is it possible?

**Comment by btry** [comment]
Hi

Just use the other setting in the 2nd dropdown :

![image](https://user-images.githubusercontent.com/14139801/128910141-b811d7cd-6ba3-4ac1-977b-1959f5dc8889.png)

For now, the plugin can link your generated ticket to an existing ticket set by the designer of the form. It is not possible to let the requester choose his ticket. This is a known limitation.

**Comment by KorvinEC** [comment]
Yes, but I need it to be chosen in the form, when user selects avilable tickets, is it possible?

**Comment by btry** [comment]
Hi

This feature requires a development. If you're interested to sponsor this feature, you may get a quote by sending a request here: https://portal.glpi-network.com/plugins/formcreator/front/formdisplay.php?id=15 

You may mention a link to this issue.
